ReactWindows/ReactNative.Shared/Views/Text/TextDecorationLine.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.Serialization;
+
+namespace ReactNative.Views.Text
+{
+    /// <summary>
+    /// TextDecorationLine values.
+    /// </summary>
+    public enum TextDecorationLine
+    {
+        /// <summary>
+        /// Text has no line decoration.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Text is Underlined.
+        /// </summary>
+        Underline,
+        /// <summary>
+        /// Text is Stroke out.
+        /// </summary>
+        LineThrough,
+        /// <summary>
+        /// Text is both Underlined and Stroke out.
+        /// </summary>
+        [EnumMember(Value = "underline line-through")]
+        UnderlineLineThrough
```

Comment:
I personally vote for enum instead of string, if there is a limited number of variants, because it 
makes corresponding code strongly-typed.